src/common/utility.ts

```diff
@@ -143,6 +143,14 @@ export class Utility {
         return null;
     }
 
+    public static async copyTemplateFile(srcPath: string, srcFile: string, targetPath: string, mapObj: Map<string, string>) {

```

Comment:
better to name "srcFile" as filename, since the file name is shared in both source and target